myapp/app/views/tasks/index.html.erb

```diff
@@ -12,15 +12,16 @@
                 tasks_path(sort_key: column[:name], sort_direction: column[:sort_direction]) %>
         </th>
       <% end %>
-      <th colspan="2"></th>
+      <th colspan="3"></th>
     </tr>
   </thead>
   <tbody>
     <% @tasks.each do |task| %>
       <tr>
-        <% task.attributes.values.each do |v| %>
+        <% task.attributes.except("description").values.each do |v| %>
           <th><%= v %></th>
         <% end %>
+        <th><%= link_to I18n.t('action.show'), task_path(task) %></th>
```

Comment:
I hid the description data from `task list page` and moved it  to `task show page`.
So added the link to `show page` in `list page` so that user can view the description of each task.